Online-Messaging-App/src/app/home/header/header.component.ts

```diff
@@ -106,6 +111,14 @@ export class HeaderComponent implements OnInit {
         this.open = !this.open;
     }
 
+    switchDisplay(value: string): void {
+        this.switchEvent.emit(value);
+
+        if (value === "profile") {
```

Comment:
use the constant you have on line 53 here

Reply:
done